src/pages/data-service-details-page/api-usage-instruction/__snapshots__/api-usage-instruction.component.test.jsx.snap

```diff
@@ -201,6 +202,7 @@ exports[`should render api-ApiUsageInstruction correctly apiSpecUrl is not set 2
       <a
         className="mr-5"
         href="https://raw.githubusercontent.com/brreg/openAPI/master/specs/api-cat.json"
+        key="https://raw.githubusercontent.com/brreg/openAPI/master/specs/api-cat.json-0"
```

Comment:
Kva er det vi driver med her, eigentlig? Eg forstår ikkje heilt kvifor vi må hardkode referanser til api-spekker i fdk-portal. Dette er forøvrig ein spesifikasjon som ikkje lenger skal ligge på den adressa.